lib/plugins/prefixer.js

```diff
@@ -1,15 +1,18 @@
 'use strict';
 
 var htmlAutoprefixer = require('html-autoprefixer');
+var postcssSafeParser = require('postcss-safe-parser');
 
-function PrefixPlugin(options) {
-  this.options = options || {};
+function PrefixPlugin(cheerioOptions, postcssOptions) {
+  this.cheerioOptions = cheerioOptions || {};
+  this.postcssOptions = postcssOptions || {};
+  this.postcssOptions.parser = this.postcssOptions.parser || postcssSafeParser;
```

Comment:
Nice. Love this. 

Only thing to keep in mind for all of us is if @M-J-Robbins has any crazy tricks that https://github.com/postcss/postcss-safe-parser ends up nullifying by cleaning up a CSS 'trick'.